app/models/lesson.rb

```diff
@@ -1,4 +1,6 @@
 class Lesson < ApplicationRecord
   belongs_to :user
   belongs_to :category
+  has_many :answers
+  has_many :words ,through: :answers
```

Comment:
Look at your COMMA ( , ).
It should be `has_many :words, through: :answers`, not like this `has_many :words ,through: :answers`